### ISSUE TITLE
Please help to clarify "authors" in 1.1

### ISSUE THREAD

**Issue by lianqi** [w3ctag/ethical-web-principles#141]
Hi TAG, when translating this document into Chinese, I find more clarification is needed to translate the term "authors" more accurately. 

In 1.1, it says "_Spec developers, **authors**, and reviewers can use it to guide their thinking._ " And there is another sentence mentioning this word in the following paragraph, saying "_For actionable guidance applicable to spec editors, **site authors**, and others designing and building parts of the web platform_". Therefore, it seems the "authors" in the first paragraph of 1.1 probably means "site authors". 

Different documents/groups in W3C have different concepts/ways of use for "author".  If the terms such as "author" and "developer" appears alone, it can be confusing about which group of people it refers to. 

Here are some examples. 

- In [Process Document](https://www.w3.org/policies/process/), it is used as "authors of the specification". 
- In A11Y, it is used at content author, see more in [ATAG2.0](https://www.w3.org/TR/ATAG20/#glossary). 
- In [I18N](https://www.w3.org/International/talks/1810-paris/index.html#developer_support), it is used as content/website author. And the same seems to apply to CSS WG as well.

**Comment by jyasskin** [comment]
That sentence would read oddly if it were "spec developers, website authors, and (what?) reviewers". I think they're all supposed to be attached to "spec". But everywhere else that we use "authors", we mean website authors, so we should change the term in that sentence. It's a little odd to distinguish spec developers from spec authors, since the people developing a specification also tend to write bits of it. Maybe "editors"? I'll send a PR doing that, and the EWP editors can check my understanding.

**Comment by lianqi** [comment]
@jyasskin Thanks a lot for the reply! Will wait for further clarification. 
Also, since I've got your attention, here is another clarificaton request. In 2.4, there is a sentence, "**Our specifications and websites** are well internationalized, provide support for language and cultural adaptation, and support localization, so that our work is accessible to all users, regardless of language, writing system, or culture. " Our specifications" is not difficult to understand. What exactly does **"our websites"** mean here? W3C website?

**Comment by jyasskin** [comment]
(I've pulled the second clarification into a new issue, so that we can figure them out independently.)

**Comment by torgo** [comment]
Yes agreed in this case I think we mean "spec authors" - that is, people writing the specification text, not website authors.
